curation_concerns-models/lib/curation_concerns/permissions.rb

```diff
@@ -0,0 +1,9 @@
+module CurationConcerns
+  module Permissions
```

Comment:
I had namespaced these in `CurationConcerns::File::Permissions`.  You're right to put them in `CurationConcerns::File::Permissions`.  Leave these ones, delete `curation_concerns/file/permissions.rb` and `curation_concerns/file/permissions` directory, and update any existing references to `CurationConcerns::File::Permissions` so that they use the new namespace.